src/logic/safe/store/actions/createTransaction.ts

```diff
@@ -169,11 +169,17 @@ export class TxSender {
       return
     }
 
-    const contractErrorMessage = await getContractErrorMessage({
-      safeInstance,
-      from,
-      data: executeData,
-    })
+    let contractErrorMessage
+    try {
+      contractErrorMessage = await getContractErrorMessage({
+        safeInstance,
+        from,
+        data: executeData,
+      })
+    } catch (err) {
+      logError(Errors._817, err.message)
```

Comment:
Here you should just call `err.log()` because it's already an instance of the CodedExpection.

Reply:
Same goes for the other instances of `logError` inside the TxSender class right?